packages-exp/auth-exp/src/model/user.ts

```diff
@@ -34,16 +34,16 @@ export interface UserParameters {
   auth: Auth;
   stsTokenManager: StsTokenManager;
 
-  displayName?: string;
-  email?: string;
-  phoneNumber?: string;
-  photoURL?: string;
-  isAnonymous?: boolean;
-  emailVerified?: boolean;
-  tenantId?: string;
+  displayName?: string | null;
```

Comment:
we really should figure out this undefined | null situation at some point...

Reply:
Agreed :P